go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/containers/image/v5 v5.17.0
 	github.com/containers/ocicrypt v1.1.2
 	github.com/containers/psgo v1.7.1
-	github.com/containers/storage v1.37.0
+	github.com/containers/storage v1.37.1-0.20220323114520-095fcc2bfaa8
```

Comment:
I'd prefer to cut a v1.37 and vendor that over vendoring a commit.

Reply:
I agree.

Reply:
Done